Source/WebKit/Platform/IPC/cocoa/ConnectionCocoa.mm

```diff
@@ -169,8 +169,9 @@ static void clearNoSenderNotifications(mach_port_t port)
     requestNoSenderNotifications(port, MACH_PORT_NULL);
 }
 
-bool Connection::open()
+bool Connection::open(Client& client)
 {
+    m_client = &client;
```

Comment:
Should we assert that m_client is nullptr before this assignment?

Reply:
Done